### DISCUSSION TITLE
Why I can't run this(at least with webui)?

### DISCUSSION THREAD

**Discussion by Dedsd** [haugene/docker-transmission-openvpn#2165] [Q&A] [answered]
Hi! I tried running in a pi but I couldn't get it to work. Checked the network with `ip -r`(192.169.0.0/24) and put the correct credentials of openvpn account. After a fill seconds checking `(health: starting)` the container stop working and restart(since I used --restart unless-stopped).  When I try to run again I get `The container name transmission-openvpn is already in use by container [container id]`. But, when I try to exec in the container I get an output that the container is not running. Anyone know what I'm doing wrong to execute?

**Comment by pkishino** [comment]
Bro, if you want help you might want to provide some more details like config/docker setup /run command etc..
Check your docker containers, you can’t create two with the same name..

**Reply by Dedsd** [reply]
> But, when I try to exec in the container I get an output that the container is not running.

**Reply by pkishino** [reply]
Not inside the container.. on your host.
Docker ps

**Reply by Dedsd** [reply]
Exec to enter in the container. Docker ps appear (health: starting) and after a fill seconds it restart because of --restart unless-stopped.

**Reply by pkishino** [reply]
Yes, so obviously if you try to run the run command again this will fail as you already have an instance..
Now, without logs can’t see why it is restarting.
Please check the wiki for more Troubleshooting

**Reply by Dedsd** [reply]
How can I see the logs? It is not running.

**Reply by pkishino** [reply]
docker logs “container name”
This will show the logs from the container in whatever state it is

**Reply by Dedsd** [reply]
Here we go:
```
Starting container with revision: 20877f1b168b6ff27fc58aeef40756572c562d47
Creating TUN device /dev/net/tun
Using OpenVPN provider: PIA
Running with VPN_CONFIG_SOURCE auto
Provider PIA has a bundled setup script. Defaulting to internal config
Executing setup script for PIA
Downloading OpenVPN config bundle openvpn into temporary file /tmp/tmp.Xl4MAiLGch
Extract OpenVPN config bundle into PIA directory /etc/openvpn/pia
Starting OpenVPN using config france.ovpn
Modifying /etc/openvpn/pia/france.ovpn for best behaviour in this container
Modification: Point auth-user-pass option to the username/password file
Modification: Change ca certificate path
Modification: Change ping options
Modification: Update/set resolv-retry to 15 seconds
Modification: Change tls-crypt keyfile path
Modification: Set output verbosity to 3
Modification: Remap SIGUSR1 signal to SIGTERM, avoid OpenVPN restart loop
Setting OpenVPN credentials...
adding route to local network 192.168.0.0/24 via 172.17.0.1 dev eth0
Mon Jan 24 00:55:10 2022 OpenVPN 2.4.7 arm-unknown-linux-gnueabihf [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [PKCS11] [MH/PKTINFO] [AEAD] built on Jul 19 2021
Mon Jan 24 00:55:10 2022 library versions: OpenSSL 1.1.1f  31 Mar 2020, LZO 2.10
Mon Jan 24 00:55:10 2022 NOTE: the current --script-security setting may allow this configuration to call user-defined scripts
Mon Jan 24 00:55:10 2022 TCP/UDP: Preserving recently used remote address: [AF_INET]191.101.217.32:1198
Mon Jan 24 00:55:10 2022 Socket Buffers: R=[180224->180224] S=[180224->180224]
Mon Jan 24 00:55:10 2022 UDP link local: (not bound)
Mon Jan 24 00:55:10 2022 UDP link remote: [AF_INET]xxx-someip
Mon Jan 24 00:55:10 2022 TLS: Initial packet from [AF_INET]xxx-someip, sid=3727f218 660cab4a
Mon Jan 24 00:55:10 2022 WARNING: this configuration may cache passwords in memory -- use the auth-nocache option to prevent this
Mon Jan 24 00:55:11 2022 VERIFY OK: depth=1, C=US, ST=CA, L=LosAngeles, O=Private Internet Access, OU=Private Internet Access, CN=Private Internet Access, name=Private Internet Access, emailAddress=secure@privateinternetaccess.com
Mon Jan 24 00:55:11 2022 VERIFY KU OK
Mon Jan 24 00:55:11 2022 Validating certificate extended key usage
Mon Jan 24 00:55:11 2022 ++ Certificate has EKU (str) TLS Web Server Authentication, expects TLS Web Server Authentication
Mon Jan 24 00:55:11 2022 VERIFY EKU OK
Mon Jan 24 00:55:11 2022 VERIFY OK: depth=0, C=US, ST=CA, L=LosAngeles, O=Private Internet Access, OU=Private Internet Access, CN=paris410, name=paris410
Mon Jan 24 00:55:11 2022 Control Channel: TLSv1.3, cipher TLSv1.3 TLS_AES_256_GCM_SHA384, 2048 bit RSA
Mon Jan 24 00:55:11 2022 [paris410] Peer Connection Initiated with [AF_INET]xxx-someip
Mon Jan 24 00:55:12 2022 SENT CONTROL [paris410]: 'PUSH_REQUEST' (status=1)
Mon Jan 24 00:55:12 2022 AUTH: Received control message: AUTH_FAILED
Mon Jan 24 00:55:12 2022 SIGTERM[soft,auth-failure] received, process exiting
```

**Reply by pkishino** [reply]
See bottom.. Auth failed means your credentials are bad or possibly contain special characters.. check wiki

**Reply by Dedsd** [reply]
My user have to be of openVPN cloud right? And if I can logging in to then I can use in transmission-openvpn?

**Comment by Dedsd** [comment]
@pkishino Oh, sorry:
Plataform: Debian arm64
Docker run:
```
 docker run --cap-add=NET_ADMIN -d --name transmissionvpn -v /home/pi/transmissionvpn/:/data -e OPENVPN_PROVIDER=PIA -e OPENVPN_CONFIG=france -e OPENVPN_USERNAME=myuser -e OPENVPN_PASSWORD=mypasswd -e LOCAL_NETWORK=192.168.0.0/24 --log-driver json-file --log-opt max-size=10m -p 3237:3237 haugene/transmission-openvpn
```
Openvpn account: https://openvpn.net/

**Comment by pkishino** [comment]
Huh? You have specified PIA as provider.. you need an account with them.
Openvpn cloud I don’t know what that is..
This container requires some VPN provider account

**Reply by Dedsd** [reply]
Do you have some suggestion of account that you use?

**Comment by pkishino** [comment]
I can recommend PrivateVPN, but depends on your location etc etc.. PIA is
often used as well.

**Reply by Dedsd** [reply]
Can be of protonvpn?

**Reply by Dedsd** [reply]
Oh, I found https://github.com/haugene/vpn-configs-contrib . But still, my openvpn credentials are correct https://protonvpn.com/support/vpn-login/, without any client already connected to the vpn and no special characters..

**Comment by Dedsd** [comment]
Hi @pkishino I seemed to fix the problem. However i'm running in a pi and when I do wget http://localhost:9091/ it founds a website, since the os of the pi that i'm running doesn't have a gui I can't access transmission with http://[pi ip]:9091/ . Any idea/solution?

**Comment by Dedsd** [comment, marked as answer]
Reinstalled my os and it worked.
